ui/src/CaseDetails.js

```diff
@@ -61,74 +73,99 @@ class CaseDetails extends Component {
     }
   }
 
-  onDeactivate = async (qid) => {
-    const response = await fetch('/deactivateUac/' + qid)
-
-    if (response.ok) {
-      // NOTE: UIs don't play nice with async stuff... we have no idea if/when the UAC will actually be dectivated... go figure
-      await new Promise(r => setTimeout(r, 1000)) // This sleep ought to work... ish. There's no better way to do this
-
-      // Now, refresh the data so we can see that the event and that the UAC is deactivated
-      this.getAllBackendData()
-    }
+  onDeactivate = (qid) => {
+    fetch('/deactivateUac/' + qid)
   }
 
   render() {
     const caseEvents = this.state.events.map((event, index) => (
-      <TableRow key={index}>
-        <TableCell component="th" scope="row">
-          {event.eventDate}
-        </TableCell>
-        <TableCell component="th" scope="row">
-          {event.eventDescription}
-        </TableCell>
-        <TableCell component="th" scope="row">
-          {event.eventSource}
-        </TableCell>
-      </TableRow>
+        <TableRow key={index}>
```

Comment:
reformatting making it a bit tricky to review

Reply:
I think I need to try and get the VSCode formatting in Webstorm to stop this happening again. Will try and merge in these bits from main now